### PR TITLE
more clarifications and responses

### PR DESCRIPTION
**Pull request by grahamgower**
I think this takes care of the largest changes.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -166,15 +166,18 @@ \section*{Introduction}
 simulators~\citep[e.g.,][]{ewing2010msms,chen2009fast,staab2015scrm}.
 However, this conciseness means that models of even intermediate complexity
 are difficult for humans to understand, making errors likely.
-Input parameter file formats for simulators allow the model specification
+APIs are more verbose, but require a substantial time investment to
+learn, and are tied to a specific tool, so this knowledge is not portable
```
Comment (jeromekelleher) [inline paper.tex]: I feel like there's one too many commas here, but can't decide which one should go?

File: paper.tex [hunk 2]
```diff
@@ -322,14 +324,38 @@ \subsection*{Example: an isolation with migration model}
...
-        (B) A visual representation of the model using \texttt{demesdraw}.
+        (B) A visual representation of the model using \demesdraw.
         The same model in the Machine Data Model form is provided in
         Figure~\ref{fig:IM-MDM}.
         The YAML descriptions in (A) and Figure~\ref{fig:IM-MDM} correspond
         exactly to a JSON description, but are much more human-readable.
     }
 \end{figure}
 
+In Figure \ref{fig:IM} we provide an example model in Demes YAML format.
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
In Figure \ref{fig:IM} we provide an example isolation-with-migration model in Demes YAML format.
```

File: paper.tex [hunk 3]
```diff
@@ -322,14 +324,38 @@ \subsection*{Example: an isolation with migration model}
...
+grows exponentially from 1000 individuals at 50 generations ago to 3000
+individuals at time 0 (the present).
+The migration section lists one migration stanza, between demes X and Y.
+This migration stanza doesn't indicate a source or destination deme,
+so the migration is symmetric. No migration times are specified, so migrations
+occur continuously at the given rate during the time interval over which
+both demes exist (from 100 generations ago until the present).
+We do not attempt a detailed explanation of all Demes features here,
+and readers are instead directed to the tutorial included
+in the online Demes specification
```
Comment (jeromekelleher) [inline paper.tex]:
```suggestion
and readers are instead directed to the tutorial and detailed specification 
in the online documentation
```

File: response-to-reviewers.tex [hunk 4]
```diff
@@ -150,15 +158,19 @@ \section*{Response to the reviewers}
 \end{point}
 \begin{reply}
 
-    Random thoughts from KT:
+We thank the reviewer for bringing this omission to our attention.
+It's important that different forward simulators use the same strategy
+for converting from times in the past (as used by Demes) into their
+corresponding values when using a forwards-time convention.
+The trick is to find the oldest non-infinite time in the model
```
Comment (jeromekelleher) [inline response-to-reviewers.tex]: s/trick/technique

"trick" could be misinterpreted by cranky reviewer

File: response-to-reviewers.tex [hunk 5]
```diff
@@ -150,15 +158,19 @@ \section*{Response to the reviewers}
...
+and use this as the forwards-time reference point, suitably adjusted
+for a burn-in. This same trick is used by DemesDraw for the model diagrams.
+We have added a section to the specification detailing how this
+conversion should be done
+(\url{https://github.com/popsim-consortium/demes-spec/pull/159}),
+and added a brief description to the "Time units" section in the manuscript's
```
Comment (jeromekelleher) [inline response-to-reviewers.tex]:
```suggestion
and added a brief description to the ``Time units'' section in the manuscript's
```

File: response-to-reviewers.tex [hunk 6]
```diff
@@ -426,14 +452,28 @@ \section*{Response to the reviewers}
...
+deme and have offspring in a "dest" deme),
+epoch ordering (from the past towards the present),
+and a deme must be listed in the input file before its
+descendant demes.
+This latter requirement means that older demes are listed first and that
+demes are topologically sorted according to ancestry relationships.
+Most non-trivial models of biological relevance are treelike with one
+"root" deme that stretches back in time towards infinity.
+The ordering in Demes naturally aligns with how rooted trees are typically drawn:
+top-down as in computer science (\ggcomment{cite Knuth, vol 1, section 2.3}),
```
Comment (jeromekelleher) [inline response-to-reviewers.tex:466]: Booya, Knuth citation!

**Review by jeromekelleher — APPROVED** [review]
Fabulous, thanks @grahamgower! Some suggestions to take or leave.

**Comment by grahamgower** [comment]
Thanks @jeromekelleher. Suggestions taken.

**Pull request merged**